src/content/getting-started/designers/designers.md

```diff
@@ -47,17 +45,16 @@ The Carbon Design System team often makes improvements to the Design Kit. We tra
 	- To update to the latest Carbon library version you will have to download the newest version of the Carbon Design Kit each time it is released and replace the previous version.
 	- Follow the specific [instructions](https://github.com/IBM/carbon-design-kit/wiki/Sketch-Libraries-Overview) on the wiki to ensure a smooth transition.
 
-## Use with Box Drive
+### Sync via Box Drive
 
-*For IBM internal users only.* Open source users should use the download from GitHub method. Using the kit through Box Drive enables automatic updates to your Carbon Design Kit library without having to manually reload down the kit everytime as in the download from GitHub method.
+_For IBM internal users only._ Using Box Drive sync will ensure that your design kit file will always be up to date, eliminating the need to manually download new versions as they are released.
 
-1. Follow the instructions on [IBM Digital Design](https://www.ibm.com/standards/web/design-kit/) to get the IBM Design kit through Box Drive.
-- Once you have installed and gained access to the `IBM Design kit`, navigate to the `product design` folder. Select the version of Carbon you'd like to use to find the `Carbon Design Kit` file.
-- When updates are available to the kit library a purple `Library Update` badge will appear in the top right corner of your Sketch file.
+1. Follow the instructions on [IBM Digital Design](https://www.ibm.com/standards/web/design-kit/) to get the design kit through Box Drive.
+- Once you have installed and gained access to the `IBM Design Kit`, navigate to the `product design` folder. Select the correct version of Carbon to find the right Carbon Design Kit file.
```

Comment:
there's not a "correct" version. Some designers might still be using v9 after our V10 release because their development team hasn't upgraded yet.

Maybe something like this:
"Select a version Carbon you are using then the Carbon kit file will be in that folder."

Reply:
revised to say "desired" instead of "correct"